src/store/filestore.rs

```diff
@@ -11,36 +12,29 @@ use std::{
     fs,
     path::{Path, PathBuf},
 };
-
-use super::{ModelID, ModelInfo, Store};
-
-const SPEC_FILE_NAME: &str = "spec.yaml";
-
-#[derive(PartialEq, Eq, PartialOrd, Ord)]
-struct NameVerTreeKey {
-    name: String,
-    version: String,
-}
-
-/// Local storage system for orca items implmenting store
+/// Support for a storage backend on a local filesystem directory.
 #[derive(Debug)]
 pub struct LocalFileStore {
+    /// A local path to a directory where store will be located.
     directory: PathBuf,
 }
 
 impl Store for LocalFileStore {
     fn save_pod(&self, pod: &Pod) -> Result<()> {
-        self.save_model(pod, &pod.hash, pod.annotation.as_ref())
+        self.save_model(
```

Comment:
This doesn't allow saving without an annotation, making annotations more of a requirement. Perhaps we need to discuss and decide on how optional is annotation

Reply:
I agree. I implemented this way to open up discussion around this. Sometimes it is easier to see and have something in front of you to help the discussions. See my [comment](https://github.com/walkerlab/orcapod/pull/42#discussion_r1827652408) here for some background.